install.bat

```diff
@@ -0,0 +1,168 @@
+    MKDIR !firefoxPath!
+    
+    ECHO Building firefox installer version %version% language !lang!...
+    
+    REM Downloading firefox setup.exe from Mozilla
+    SET fileURL=http://ftp.mozilla.org/pub/firefox/releases/%version%/win64/!lang!/Firefox%%20Setup%%20%version%.exe
+    SET fileLocation=!firefoxPath!\firefox_%version%_!lang!.exe
+    cscript.exe src\downloadfile.vbs
+    
+    REM Check that setup.exe downloaded successfully
+    IF NOT exist !fileLocation! (
+        ECHO Could not download Firefox. Check http://ftp.mozilla.org/pub/firefox/releases/ for valid versions and languages.
+        GOTO End
+    )
+    
+    REM Extracting files from Firefox_setup.exe
+    %sevenzip% x -y !fileLocation! -o!buildPath!
+    
+    SET fileURL=
+    SET fileLocation=
+    
+    REM Copy new files from src to build
+    COPY /Y src\core\custom-config.cfg !buildPath!\core\custom-config.cfg
+    IF NOT exist !buildPath!\core\defaults\pref {
+        MKDIR !buildPath!\core\defaults\pref\
+    }
+    COPY /Y src\core\pref\local-settings.js !buildPath!\core\defaults\pref\local-settings.js
+    COPY /Y src\core\pref\user.js !buildPath!\core\defaults\pref\user.js
+    COPY /Y src\core\pref\channel-prefs.js !buildPath!\core\defaults\pref\channel-prefs.js
+    
+    REM Add each file to the uninstall file
+    SET uninstallFile=!buildPath!\core\precomplete
+    ECHO remove "custom-config.cfg" >> !uninstallFile!
+    ECHO remove "defaults/pref/local-settings.js" >> !uninstallFile!
+    ECHO remove "defaults/pref/user.js" >> !uninstallFile!
+    ECHO remove "defaults/pref/channel-prefs.js" >> !uninstallFile!
+    ECHO rmdir "defaults/pref/" >> !uninstallFile!
+    ECHO rmdir "defaults/" >> !uninstallFile!
+
+    REM Copy landing page for language to core
+    REM TODO: Uncomment following lines to add landing page
+    REM COPY /Y src\landingpage\en-US\uproxy.html !buildPath!\core\uproxy.html
+    REM ECHO remove "uproxy.html" >> !uninstallFile!
+    
+    REM Copy all distribution files last
+    MKDIR !buildPath!\core\distribution
+    XCOPY /E src\core\distribution !buildPath!\core\distribution
+    
+    REM Extract uproxy.xpi to extension directory
+    %sevenzip% x -y %parent%extension\uproxy.xpi -o!buildPath!\core\distribution\extensions\jid1-uTe1Bgrsb76jSA@jetpack
+
+    REM Compress files into build\%lang%\app.7z
+    CD !buildPath!
+    %sevenzip% a -r -t7z app.7z -mx -m0=BCJ2 -m1=LZMA:d24 -m2=LZMA:d19 -m3=LZMA:d19 -mb0:1 -mb0s1:2 -mb0s2:3
+    CD %parent%
+    
+    REM Repackage app.7z into custom exe
+    SET distFile=Firefox_with_uProxy_%version%_!lang!.exe
+    COPY /B src\7zSD.sfx+src\app.tag+!buildPath!\app.7z !distPath!\!distFile!
+    ECHO Completed build for !distFile!
+    
+    REM Sign final exe with signtool.exe
+    REM TODO: Uncomment following line to sign package when we have a cert 
+    REM %signtool% sign /v /n uProxy /p [password] /f [uProxy.pfx] /t http://timestamp.verisign.com/scripts/timstamp.dll !distPath!\!distFile!
```

Comment:
Updated the code signing line so that it actually signs the installer with our cert.